vividus-to-xray-exporter/src/main/java/org/vividus/xray/configuration/XrayExporterOptions.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.xray.configuration;
+
+import java.nio.file.Path;
+import java.util.List;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties("xray-exporter")
+public class XrayExporterOptions
+{
+    private Path sourceDirectory;
+    private String projectKey;
+    private List<String> editableStatuses;
+    private String skipMarker;
```

Comment:
why is it needed from business point of view?

Reply:
discussed to clarify skipping of test cases while import